app/views/posts/show.html.erb

```diff
@@ -1 +1,18 @@
-<h1>Single Post will be shows here</h1>
\ No newline at end of file
+<div>
+  <div>
+   <div>
+     <%= link_to "Post ##{1} by #{@post.author.name}", user_post_path(@post.author, @post) %>
+     <span>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></span>
+   </div>
+   <div><%= @post.text %></div>
+</div>
+
+<% if @post.comments == [] %> 
+  <p>This post has no comments yet!</p>
+<% else %>
+    <% @post.comments.each do |comment| %>
+      <ul>
+        <li><%= @post.author.name %>: <%= comment.text %></li>
+      </ul>
+    <% end %>
+<% end %>
```

Comment:
- [ ] Kindly make sure the posts show the page matches the layout shown in the wireframe below.
- Your design:
  ![image](https://github.com/Aqurds/coding-thought/assets/104968236/986ab9cd-4a5a-4c7d-93ae-ad958f8205dc)
- Wireframe:
  ![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAACJTIBN6RLQPNUX3EO63KZ6SZKE4YKA)

Reply:
Done